spec/order_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'order'
+
+describe Order do
+
+  let(:dish1){double(:dish1, name: "Dishy1", price: 3.50)}
+  let(:dish2){double(:dish2, name: "Dishy2", price: 5.00)}
+  let(:dish3){double(:dish3, name: "Dishy3", price: 4.99)}
+  let(:menu1){double(:menu1, select: dish1)}
+  let(:menu2){double(:menu2, select: dish2)}
+  let(:menu3){double(:menu3, select: dish3)}
+
+
+  describe '#add_to_basket' do
+    it 'calls the menu#select method' do
+      expect(menu1).to receive(:select)
+      subject.add_to_basket(menu1, 1, 2)
+    end
+
+    it 'given the menu, dish number and quantity adds the dish/dishes to the order' do
+      subject.add_to_basket(menu1, 1, 2)
+      expect( subject.basket.count(dish1) ).to eq 2
+    end
+  end
+
+  describe '#summary' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.